components/filter-dropdown/filter-dropdown.js

```diff
@@ -198,6 +267,19 @@ class D2LLabsFilterDropdown extends mixinBehaviors([LocalizeBehavior], PolymerEl
 		this._hasFooter = e.target.assignedNodes().length !== 0;
 	}
 
+	_handleTabSelected(e) {
+		e.stopPropagation();
+		if (!this._useNewStructure) return;
+
+		// Find the corresponding category and dispatch the selection event
+		const tabId = e.target.id;
+		const category = this.querySelector(`d2l-labs-filter-dropdown-category[key="${tabId}"]`);
+
+		if (category) {
+			category._dispatchSelected();
```

Comment:
Maintaining how this worked previously; within filter-dropdown-category it listens for tab-panel-selected and runs _dispatchSelected